src/components/subHeader/SubHeader.jsx

```diff
@@ -0,0 +1,88 @@
+import "./subHeader.styles.css";
+
+import restaurantImage from "./rest-img.jpeg";
+
+export const SubHeader = ({ onClick, onChange }) => {
+  return (
+    <section className="restaurant__section">
+      <img
+        src={restaurantImage}
+        alt="restaurant"
+        className="restaurant__image"
+      />
+      <div className="restaurant__info">
+        <p className="restaurant__info--name">Kitchens of Punjab</p>
+        <p className="restaurant__info--type">Punjabi, North Indian</p>
+        <p className="restaurant__info--address">
+          Bilekahalli, BTM - Bangalore | Change Outlet
+          <i className="fa-solid fa-angle-down"></i>
+        </p>
+        <div className="restaurants__highlights">
+          <div>
+            <p className="restaurants__highlights__features">
+              <i className="fa-solid fa-star"></i> 4.1
+            </p>
+            <p className="restaurants__highlights__text">100+ Ratings</p>
+          </div>
+          <div>
+            <p className="restaurants__highlights__features">
+              <i className="fa-solid fa-motorcycle"></i> 74min
+            </p>
+            <p className="restaurants__highlights__text">Delivery Time</p>
+          </div>
+          <div>
+            <p className="restaurants__highlights__features">&#x20B9; 400</p>
+            <p className="restaurants__highlights__text">Cost for two</p>
+          </div>
+        </div>
+
+        <div className="restaurant__info__input-fields">
+          <div className="restaurant__info__input-group">
+            <input
+              type="text"
+              id="restaurant__info__search"
+              className="restaurant__info__search"
+              placeholder="Search for dishes..."
+              onChange={(e) => onChange(e)}
```

Comment:
no need to wrap the function just pass onChange directly

Reply:
done